content/docs/user-guide/contributing/core.md

```diff
@@ -44,7 +44,10 @@ $ git clone git@github.com:<username>/dvc.git
 
 Make sure that you have Python 3 installed. Version 3.6 or higher is required to
 run style checkers on pre-commit. On MacOS, we recommend using `brew` to install
-Python.
+Python. For Windows, we recommend an official
```

Comment:
seems too specific to include this tbh

Reply:
Hmmm OK. But it was really annoying having to figure all this out haha. Windows...

I'm just leaving "For Windows, we recommend an official python.org release." here, since we already had a similar rec for MacOS. Or should we remove both?

Reply:
Yeah, we do for Mac because it has a pre-installed Python 2 - which is very old and incompatible and should not be used at all

Reply:
Good point. But that's the thing though, Window also comes with built in `python` command, but it's an alias that go to the Microsoft Store Python installer. But if you install it that way, it doesn't work on Git Bash or other special shells.

Reply:
I saw we keep both (see latest version) or remove both, because the Python version that comes on MacOS is 2 anyway, from what I remember, and we already specify that we require py 3.

Reply:
Merged this as is now. Lmk if you prefer that I remove the notes and I'll send a quick PR for that. Thanks